.gitignore

```diff
@@ -38,6 +38,7 @@ run
 *#
 .#*
 .dir-locals.el
+.projectile
```

Comment:
this feels like something that should go in a global gitignore, but I don't really care all that much about `.gitignore` hygiene :)